D-OS Save Editor/SE/SaveEditor.xaml.cs

```diff
@@ -47,6 +49,29 @@ public SaveEditor(Savegame savegame)
         {
             InitializeComponent();
 
+            ShowMeta = savegame.Meta;
+            // check number of game version and inform user
+            if (ShowMeta.GameVersion.Count > 1)
+            {
+                string formatedVersions = "";
+                foreach (string gameVersion in ShowMeta.GameVersion)
+                {
+                    formatedVersions += "\n  " + gameVersion;
+                }
+                MessageBox.Show("DOS EE Editor found " + ShowMeta.GameVersion.Count + " game versions for this savegame\nDOS EE Editor and this modified savegame may be unstable: keep play on latest version you have.\nGame Version:" + formatedVersions, "Too many game version", MessageBoxButton.OK, MessageBoxImage.Warning);
+            }
+
+            if (ShowMeta.ModsName.Count > 1)
+            {
+                string formatedMods = "";
+                foreach (string modName in ShowMeta.ModsName)
+                {
+                    if (modName != "Shared")
+                        formatedMods += "\n  " + modName;
+                }
+                MessageBox.Show("DOS EE Editor found " + (ShowMeta.ModsName.Count - 1) + " mod" + ((ShowMeta.ModsName.Count - 1) > 1 ? "s" : "") + " for this savegame\nDOS EE Editor and this modified savegame may be unstable.\nThe savegame will be mostly corrupted if it's edited by DOS EE Editor\nMods name:" + formatedMods, "Mod" + (ShowMeta.ModsName.Count > 1 ? "s" : "") + " found", MessageBoxButton.OK, MessageBoxImage.Warning);
```

Comment:
Is this true? 

> will be mostly corrupted

Reply:
By example, when I want add attribute point to one of my character when using Epic Encounter, the game say the save is corrupt and don't show change. I know all mods are different, but by precaution.

But I've very poor lexical in English and the word it's maybe too "strong" ? Maybe we can replace it by often ?